README.md

```diff
@@ -264,3 +264,7 @@ First, you'll need to install the virtualenv yourself:
 Also, you will need to manually register the pack if you make changes to its metadata or configuration:
 
   st2 pack register napalm
+
+## Maintainers
+Active pack maintainers with review & write repository access and expertise with NAPALM:
+* Jeremiah Millay ([@floatingstatic](https://github.com/floatingstatic)), Fastly
```

Comment:
@floatingstatic Please review if that looks good to you. Besides of that, listing any email contact here would be nice in general, but not strictly necessary.

Reply:
LGTM